app-agent/src/main.rs

```diff
@@ -28,7 +28,7 @@ fn main() {
     let args = Cli::parse();
 
     // Specifies the plugins that we want to load.
-    let plugins = static_plugins![RaplPlugin, CsvPlugin, SocketControlPlugin, PerfPlugin];
+    let plugins = static_plugins![RaplPlugin, CsvPlugin, PerfPlugin];
```

Comment:
Why is the SocketControlPlugin not added anymore ?

Reply:
I'll put it back, it didn't compile a few days ago. Thanks!